nathan-kavdi/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Lab 1</title>
+  <!-- TODO: Add a meta tag to properly set the viewport and initial scale. DONE -->
+  <meta name="viewport" content="width=device-width" />
```

Comment:
You need to set `initial-scale` in order for your viewport tag to work properly.